packages/terra-outline-table-view/tests/jest/Table.test.jsx

```diff
@@ -4,10 +4,42 @@ import OutlineTableView from '../../src/Table';
 
 describe('OutlineTableView', () => {
   // Snapshot Tests
-  it('should render a OutlineTableView', () => {
+  it('should pass through props to Table', () => {
```

Comment:
passthrough is one word, no?
```suggestion
  it('should passthrough props to Table', () => {
```